addons/project/static/src/js/project.js

```diff
@@ -18,7 +18,7 @@ KanbanRecord.include({
      * @private
      */
     _openRecord: function () {
-        if (this.modelName === 'project.project') {
+        if (this.modelName === 'project.project' && $(".o_project_kanban_boxes").length != 0) {
```

Comment:
What if record.use_tasks.raw_value is false? We will not able to open this record...

Reply:
use_tasks will be removed. So, it's not a problem.
I suggest this:
if (this.modelName === 'project.project' && this.$(".o_project_kanban_boxes a")) {

Reply:
Done.

Reply:
i have changed it to this, .length is needed:                                                                                                                                  if (this.modelName === 'project.project' && this.$(".o_project_kanban_boxes a").length) {